src/main/java/org/opentripplanner/service/vehiclepositions/VehiclePositionRepository.java

```diff
@@ -0,0 +1,29 @@
+package org.opentripplanner.service.vehiclepositions;
```

Comment:
My suggestion for the package structure:
```
o.o.routing.vehiclepositions
  Service/Repository interface
  .configure
  .service
    DefaultVPS (Service and Repository implementaion)
  .model  
```

Reply:
So basically move the the package into `o.o.routing.vehiclepositions` because the hierarchy you describe already exists.